### DISCUSSION TITLE
Allow SAML auth to be set in the organizations API

### DISCUSSION THREAD

**Discussion by anthr76** [community/community#13289] [Projects and Issues]
https://docs.github.com/en/rest/reference/orgs#update-an-organization

You cannot set SAML in the organization API. It would be useful for this functionality.

**Comment by denniseffing** [comment]
Yes, please! I was just looking for this functionality and still can't believe that it's not possible at all. Neither via REST nor GraphQL.

You _can_ actually configure SAML on enterprise level. However, this is limited to the GraphQL API using the `setEnterpriseIdentityProvider` mutation. There is no equivalent in the REST API.
